### PR TITLE
Update CreatorKit top level build system for cascoda configuration

### PR DESCRIPTION
**Pull request by abhijit-mahajani-imgtec**
Add creator-kit-1-cascoda.config for boards with cascoda ca8210.
Add creator-kit-0-cascoda.config for boards with cascoda ca8210.
Rename creator-kit.config to creator-kit-0.config
Update build makefile to build contiki for CA8210 or CC2520 depending on config file
Add creator-kit-all-cascoda.config

### CODE REVIEW

**Comment by pratik-prajapati-imgtec** [comment]
CR+1

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 J?=1
+CASCODA=0
```
Comment (mayank-sirotiya-imgtec) [inline Makefile]: please remove this

File: creator-kit-all-cascoda.config [hunk 2]
```diff
@@ -0,0 +1,14 @@
+CONFIG_TARGET_pistachio=y
+CONFIG_TARGET_pistachio_marduk=y
+CONFIG_TARGET_pistachio_marduk_marduk_ca8210=y
+CONFIG_PACKAGE_button-gateway=y
+CONFIG_PACKAGE_device-manager=y
+CONFIG_PACKAGE_webscripts=y
+CONFIG_ALL=y
```
Comment (nikhil-zinjurde-imgtec) [inline creator-kit-all-cascoda.config]: Please remove this as we don't need any project to be enabled by default for build-all configuration

File: creator-kit-all-cascoda.config [hunk 3]
```diff
@@ -0,0 +1,14 @@
+CONFIG_TARGET_pistachio=y
+CONFIG_TARGET_pistachio_marduk=y
+CONFIG_TARGET_pistachio_marduk_marduk_ca8210=y
+CONFIG_PACKAGE_button-gateway=y
+CONFIG_PACKAGE_device-manager=y
+CONFIG_PACKAGE_webscripts=y
```
Comment (nikhil-zinjurde-imgtec) [inline creator-kit-all-cascoda.config]: Please remove this as we don't need any project to be enabled by default for build-all configuration

**Comment by pratik-prajapati-imgtec** [comment]
Approved & Verified

**Pull request merged**